srcpkgs/coyim/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'coyim'
+pkgname=coyim
+version=0.3.11
+revision=1
+build_style=go
+go_import_path=github.com/coyim/$pkgname
+hostmakedepends="pkg-config"
+makedepends="gtk+3-devel"
+depends="gtk+3"
```

Comment:
This is not necessary, `xbps-src` detects the dependency when looking at the shared libaries the resulting binary links against.
https://travis-ci.org/void-linux/void-packages/jobs/577838250#L5938

Reply:
Removed it